dispatch/BESInterface.cc

```diff
@@ -32,18 +32,18 @@
 
 #include "config.h"
 
-#include <string>
-#include <sstream>
-#include <iostream>
+#include <cstdlib>
 
+#include <signal.h>
 #if HAVE_UNISTD_H
 #include <unistd.h>
 #endif
 
-using std::string;
-using std::ostringstream;
-using std::bad_alloc;
-using std::cout;
+#include <setjmp.h> // Used for the timeout processing
```

Comment:
On January 4, 2016 at 6:00:35 PM, Patrick West (notifications@github.com) wrote:

In dispatch/BESInterface.cc:

>  #if HAVE_UNISTD_H
>  #include <unistd.h>
>  #endif
>   
> -using std::string;
> -using std::ostringstream;
> -using std::bad_alloc;
> -using std::cout;
> +#include <setjmp.h> // Used for the timeout processing
should this be conditional based on if on windows system? I've seen the macros in other places but not here. Or is this a generic include?

AFAIK we don’t have a windows build for the BES. 


—
Reply to this email directly or view it on GitHub.


-- 
James Gallagher
jgallagher@opendap.org